dev/io.openliberty.microprofile.lra.1.0.internal/src/io/openliberty/microprofile/lra/internal/LraFilterComponent.java

```diff
@@ -86,38 +87,33 @@ protected void deactivate(int reason) {
     @Override
     public void installProvider(boolean clientSide, List<Object> providers, Set<String> features) {
         if (TraceComponent.isAnyTracingEnabled() && tc.isEventEnabled()) {
-            Tr.event(tc, "Attempting to register LRA filters");
+            Tr.event(tc, "Registering LRA filters");
         }
 
         if (clientSide) {
-            if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+            if (TraceComponent.isAnyTracingEnabled() && tc.isEventEnabled()) {
                 Tr.event(tc, "Registering client side filters");
             }
             ClientLRARequestFilter requestFilter = new ClientLRARequestFilter();
             providers.add(requestFilter);
             ClientLRAResponseFilter responseFilter = new ClientLRAResponseFilter();
             providers.add(responseFilter);
         } else {
-            if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+            if (TraceComponent.isAnyTracingEnabled() && tc.isEventEnabled()) {
                 Tr.event(tc, "Registering serverside side filters");
             }
-
-            ServerLRAFilter filter = null;
             try {
-                filter = new ServerLRAFilter();
+                // Rather unhelpfully, the ServerLRAFilter constructor throws 'Exception'. There isn't much we can do with
+                // that. Re-throwing should prevent the servlet being initialized, which is probably better than swallowing
+                // the exception
+                providers.add(new ServerLRAFilter());
             } catch (Exception e) {
-
-                // TODO Do something meaningful
-
-                // TODO Auto-generated catch block
-                // Do you need FFDC here? Remember FFDC instrumentation and @FFDCIgnore
-                // https://websphere.pok.ibm.com/~alpine/secure/docs/dev/API/com.ibm.ws.ras/com/ibm/ws/ffdc/annotation/FFDCIgnore.html
-                Tr.error(tc, "Couldn't register the Server filter", e);
-                return;
+                if (TraceComponent.isAnyTracingEnabled() && tc.isErrorEnabled()) {
+                    Tr.error(tc, "Exception during registration of LRA filters", e);
+                    throw new LraRuntimeException(Tr.formatMessage(tc, "LRA_CANT_REGISTER_FILTERS", e), e);
+                }
```

Comment:
Don't check `isAnyTrackingEnabled` before logging errors or throwing exceptions.

Don't need to check `isErrorEnabled()` either.